pkg/kritis/admission/admission.go

```diff
@@ -238,11 +238,12 @@ func reviewImages(images []string, ns string, pod *v1.Pod, ar *v1beta1.Admission
 		return
 	}
 	if len(isps) == 0 {
-		glog.Errorf("No ISP's found in namespace %s", ns)
-	} else {
-		glog.Infof("Found %d ISPs to review image against", len(isps))
+		glog.Errorf("No ISP's found in namespace %s, skip reviewing", ns)
```

Comment:
IMO, this should not be `Error` level log. `INFO` should be sufficient. This will make logging noisy. I understand this is coming from current Kritis code base. 

Let's refactor log level in different task so that we will be able to monitor it properly.

Reply:
Ah, yes I agree.
Not only here, we could refactor logging (level, format, etc).
As you said, let's do it in different task.